src/test/java/org/jabref/logic/cleanup/EprintCleanupTest.java

```diff
@@ -27,4 +27,23 @@ void cleanupCompleteEntry() {
 
         assertEquals(expected, input);
     }
+
+    @Test
+    void cleanupEntryWithVersionAndInstitutionAndEid() {
+        BibEntry input = new BibEntry()
+                .withField(StandardField.NOTE, "arXiv: 1503.05173")
+                .withField(StandardField.VERSION, "1")
+                .withField(StandardField.INSTITUTION, "arXiv")
+                .withField(StandardField.EID, "arXiv:1503.05173");
+
+        BibEntry expected = new BibEntry()
+                .withField(StandardField.EPRINT, "1503.05173v1")
+                .withField(StandardField.EPRINTTYPE, "arxiv")
+                .withField(StandardField.INSTITUTION, "tbd");
```

Comment:
Lets use `other-institution` as example for the **new** test.

Reply:
I have done the necessary changes, updated the EprintCleanup class to clear the institution field if it is "arXiv" else keep it otherInstitution as it is.